tests_e2e/tests/scripts/agent-update-config

```diff
@@ -20,14 +20,14 @@
 #
 set -euo pipefail
 
-AGENT_PYTHON_SCRIPT="agent-python"
-AGENT_SERVICE_SCRIPT="agent-service"
-PYTHON=$(source $AGENT_PYTHON_SCRIPT)
+PYTHON=$(get-agent-python)
```

Comment:
let's add a comment describing what this script does

Reply:
Already added a comment.